Doc/whatsnew/3.14.rst

```diff
@@ -563,6 +563,22 @@ getopt
 * Add support for returning intermixed options and non-option arguments in order.
   (Contributed by Serhiy Storchaka in :gh:`126390`.)
 
+
+heapq
+-----
+
+* The max-heap functions in :mod:`heapq` are now public:
+
+  * :func:`heapq.heapify_max`,
+  * :func:`heapq.heappush_max`,
+  * :func:`heapq.heappop_max`,
+  * :func:`heapq.heapreplace_max`
+
+  Previous underscored naming is kept for backwards compatibility.
+  Additionally, the missing function :func:`heapq.heappushpop_max` has been added to both the C and Python
+  implementations.
```

Comment:
I think we can omit mentions of the private names. People who already use them don't need this note.

(The NEWS blurb is a good place for this detail, though.)
```suggestion
* Add functions for working with max-heaps:

  * :func:`heapq.heapify_max`,
  * :func:`heapq.heappush_max`,
  * :func:`heapq.heappop_max`,
  * :func:`heapq.heapreplace_max`
  * :func:`heapq.heappushpop_max`
```